Stack/Opc.Ua.Core/Types/Utils/Utils.cs

```diff
@@ -1731,15 +1747,34 @@ public static object Clone(object value)
                 {
                     return castedObject.MemberwiseClone();
                 }
-            }           
+            }
             // copy Opc.Ua.TimeZoneDataType
             {
                 TimeZoneDataType castedObject = value as TimeZoneDataType;
                 if (castedObject != null)
                 {
                     return castedObject.MemberwiseClone();
                 }
-            }           
+            }
+            // copy Opc.Ua.LiteralOperand
+            {
+                LiteralOperand castedObject = value as LiteralOperand;
+                if (castedObject != null)
+                {
+                    return castedObject.MemberwiseClone();
+                }
+            }
+
+            //try to find the MemberwiseClone method by reflection.
+            MethodInfo memberwiseCloneMethod = type.GetMethod("MemberwiseClone", BindingFlags.Public | BindingFlags.Instance);
```

Comment:
Does it mean we can get rid of all the type compares?

Reply:
Yes, but I think it will introduce some performance penalty.
A better option would be to have the ICloneable interface back.
It was introduced now in .NET Standard 2.0